src/MoBi.Presentation/Nodes/ExpressionProfilesNode.cs

```diff
@@ -0,0 +1,19 @@
+﻿using MoBi.Assets;
+using OSPSuite.Assets;
+using OSPSuite.Presentation.Nodes;
+using OSPSuite.Utility;
+
+namespace MoBi.Presentation.Nodes
+{
+   public class ExpressionProfilesNode : AbstractNode
```

Comment:
Can we call it Folder =>  ExpressionProfileFolderNode. I think this is what we use for Folder and it's clearer than seeing the s